common/src/stack/report-system/command/report/system/conftest.py

```diff
@@ -0,0 +1,7 @@
+from pathlib import Path
+
+# Everything in "fixures" gets loaded as a plugin
+pytest_plugins = [
+	f"stack.commands.report.system.fixtures.{path.stem}" for path in Path(__file__).parent.glob("fixtures/*.py")
```

Comment:
Is there a way to make it a relative path import?  Perhaps by appending to the python path?

Reply:
```
f"fixtures.{p.stem}" for p in Path(__file__).parent.glob('fixtures/*.py')
```
_Should_ work as long as your fixtures directory has an `__init__.py` in it. If it doesn't, try `f".fixtures.{p.stem}"`

Reply:
I believe that's what I had at first and it failed to import. My fixtures directory did have a __init__.py file. I will try again to confirm.

Reply:
@caladd Neither of those options worked. @bsanders What do we get with a relative import?